styles.css

```diff
@@ -1,26 +1,25 @@
 .heading {
-    margin-left: 5%;
-    margin-top: 5%;
-  }
+  margin-left: 5%;
+  margin-top: 5%;
+}
 
-  #flex-parent{
-    align-items: center;
-    margin-top: 20px;
-    margin-left: 80px;
-    display: flex;
-  }
+#flex-parent {
+  align-items: center;
+  margin-top: 20px;
+  margin-left: 80px;
+  display: flex;
+}
 
-#num{
+#num {
   margin-left: 10px;
   width: 86px;
 }
 
-
-.hide-loader{
-  display :none;
+.hide-loader {
+  display: none;
 }
 
-.errMessage{
+.errMessage {

```

Comment:
Please choose 1 naming convention (for exmaple, camelCasing) and keep with it. using .errMessage & .hide-loader toghether is bad practice.